api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 300**
+
+* [`GET /messages`](/api/get-message): Added a parameter `message_ids` to
+  fetch messages provided a list of message ids. `message_ids` is mutually
+  exclusive to `anchor`, `num_after` and `num_before`. Note that this makes
+  all the parameters of GET request to `/messages` API endpoint optional.
```

Comment:
I would write for the prose:

> Added a new `message_ids` parameter, as an alternative method of specifying which messages to fetch.

Reply:
Changed it here and added a **changes** entry to `get` description.